### DISCUSSION TITLE
MultiPlay 에서 State 와 Animation 처리

### DISCUSSION THREAD

**Discussion by jefflim69** [naverz/zepeto-studio-kor#1252] [5. Q&A]
현재 탈것, 의자 앉기 등을 처리하고 있는데 , MultiPlay 쪽을 어떻게 처리해야 할지 좀 난감하네요 .

* 다른 Player 에게 나의 상태(앉기, 탈것에 탑승 등)을 보이도록 처리하는 방법이 있을까요 ?
  Character State 에 State를 추가하거나 하는 수정은 위험한 것 같고...

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. https://github.com/naverz/zepeto-studio-kor/discussions/1169 게시글을 참고하시면 좋을 것 같습니다 :)
